services/ingestion/etl/budg/xls/serverless.yml

```diff
@@ -0,0 +1,76 @@
+service: ingestion-etl-budg-xls
+
+plugins:
+  - serverless-webpack
+
+custom:
+  webpackIncludeModules: true
+  eubfrEnvironment: ${opt:eubfr_env, file(../../../../../config.json):eubfr_env, env:EUBFR_ENV, 'dev'}
+  bucketName: ${file(../../../../harmonized-storage/serverless.yml):custom.bucketName}
+
+package:
+  individually: true
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  timeout: 60
+  stage: ${opt:stage, file(../../../../../config.json):stage, 'dev'}
+  region: ${opt:region, file(../../../../../config.json):region, 'eu-central-1'}
+  deploymentBucket:
+    name: eubfr-${self:custom.eubfrEnvironment}-deploy
+  stackTags:
+    ENV: ${self:custom.eubfrEnvironment}
+  iamRoleStatements:
+    -  Effect: "Allow"
+       Action:
+         - "s3:PutObject"
+       Resource:
+         Fn::Join:
+           - ""
+           - - "arn:aws:s3:::"
+             - ${self:custom.bucketName}
+             - "/*"
+
+functions:
+  parseXls:
+    handler: src/events/onParseXLS.handler
+    name: ${self:provider.stage}-${self:service}-parseXls
+    memorySize: 256
+    environment:
+      BUCKET: ${self:custom.bucketName}
+    events:
```

Comment:
You could (or should) also subscribe to `${self:provider.stage}-etl-budg-xlsx` topics there ;)